src/docker.cc

```diff
@@ -21,11 +21,13 @@
 #include <boost/network/protocol/http/client.hpp>
 #include <chrono>
 
+#include "configuration.h"
```

Comment:
Is this line necessary? I only ask because it didn't exist before, and it doesn't seem like any lines added call out this as a requirement any more than the lines before this PR. Same question throughout.

Reply:
It got moved from the header file.